Overview.src.html

```diff
@@ -1042,6 +1040,12 @@ <h4 id=terminology-headers>Headers</h4>
 for the <span title=concept-response>response</span>. The list is empty unless otherwise
 specified. <span data-anolis-ref>CSP</span>
 
+<p>A <span title=concept-response>response</span> has an associated
+<dfn title=concept-response-cors-exposed-headers-list>CORS-exposed headers list</dfn>
+(a list of zero or more
+<span title=concept-header>header</span> <span title=concept-header-name>names</span>).
+The list is empty unless otherwise specified.
+
```

Comment:
Can you add a note here that says this is only used for CORS filtered responses?